lights/lights.c

```diff
@@ -159,9 +159,15 @@ static int open_lights(const struct hw_module_t *module, char const *name,
 	} while (entry = readdir(dir));
 	closedir(dir);
 
+	ALOGI("backlight path: %s\n", backlight_path);
+
 	snprintf(buffer, 1024, "%s/max_brightness", backlight_path);
 	brightness_max = read_int(buffer);
-	ALOGE("JDB: brightness_max: %ld\n", brightness_max);
+
+	if (brightness_max <= 0)
+		ALOGE("invalid max brightness: %ld\n", brightness_max);
```

Comment:
It can if e.g. there's a permission error when trying to read from the brightness file (the reader function returns -errno).